server/database/connection.js

```diff
@@ -46,34 +45,29 @@ const createTables = () => {
   const petitionTable = `CREATE TABLE IF NOT EXISTS 
```

Comment:
you could rename this to `createPetitionTableQuery`

Reply:
Querry renamed